ComposureWatch/src/Components/PlayerCards.jsx

```diff
@@ -1,6 +1,160 @@
 const PlayerCards = (props) => {
-  if (props.name == "moira") {
-    return (cardImg = <img src="src/IMG/portrait-moira.png" />);
+  const charList = [
+    {
+      name: "Moira",
+      img: "src/IMG/portrait-moira.png",
```

Comment:
You might want to look at https://vitejs.dev/guide/assets.html for handling static assets like images.

This will work when running in "dev" mode; however, if you try to build or use the `vite preview` mode, you will notice that the images are broken. This is because Vite does a lot of cool stuff with your assets (like cache busting).

I think you will need to "import" each of these assets so that Vite knows to include them in the production bundle.